tests/playwright/src/model/pages/run-image-page.ts

```diff
@@ -57,21 +62,29 @@ export class RunImagePage extends BasePage {
     await portMapping.fill(port);
   }
 
-  async startContainer(customName = '', interactive?: boolean): Promise<ContainersPage> {
+  async startContainer(customName = '', optionalParams?: ContainerInteractiveParams): Promise<void> {
     if (customName !== '') {
       await this.activateTab('Basic');
       // ToDo: improve UI side with aria-labels
       const textbox = this.page.locator(`input[type='text'][name='modalContainerName']`);
       await textbox.fill(customName);
     }
 
-    if (!interactive) {
+    if (optionalParams?.attachTerminal !== undefined) {
       // disable the checkbox in advanced tab
       await this.activateTab('Advanced');
       const checkbox = this.page.getByRole('checkbox', { name: 'Attach a pseudo terminal' });
-      await checkbox.uncheck();
+      optionalParams.attachTerminal ? await checkbox.check() : await checkbox.uncheck();
```

Comment:
Checking and unchecking should be waited and asserted on to ensure that the system waits until the checkbox is actually marked.

Please refactor this into a if-else clause where the checking/unchecking is asserted with `await playExpect(checkbox).toBeChecked()/not.toBeChecked()`

Reply:
Should be fine now.